package.json

```diff
@@ -63,5 +66,9 @@
   },
   "resolutions": {
     "axios": "^1.6.5"
+  },
+  "lint-staged": {
+    "*": "prettier --write",
+    "*.ts": "yarn lint:code --fix"
```

Comment:
can we check that this does not do lint fixes on the tests files? There are some stuff that linting changes in the tests that can break things

Reply:
it will actually lint the test files so good catch, found a workaround though so will add that in

Reply:
nice :) also make sure it picks up our lint configurations and not just lints from another set of rules

Reply:
or actually.. I would leave the linting in the pre-commit, if we have it in the pre-commit, then I do not see a reason why we should repeat it pre-push